pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -91,9 +92,14 @@ const (
 	unnamedInitContainerPrefix = "build-step-unnamed-"
 	// Name of the credential initialization container.
 	credsInit = "credential-initializer"
+	// Name of the working dir initialization container.
+	workingDirInit = "working-dir-initializer"
 )
 
 var (
+	// The container used to initialize working directories before the build runs.
+	bashWorkingDirImage = flag.String("bash-working-dir-image", "override-with-bash-working-dir:latest",
```

Comment:
The image could refer `bash` image

```go
bashWorkingDirImage = flag.String("bash-working-dir-image", "override-with-bash-noop:latest", …
```

That way we don't need to publish a new image but it can be customized at runtime :angel:

Reply:
Done.